### PR TITLE
TRUNK-4632 Dodgy - Redundant nullcheck of value known to be non-null

### PR DESCRIPTION
**Pull request by tomaszmueller**
https://issues.openmrs.org/browse/TRUNK-4632

### CODE REVIEW

File: api/src/main/java/org/openmrs/PatientIdentifierType.java [hunk 1]
```diff
@@ -44,12 +44,12 @@
 		 * Indicates that identifiers should be globally unique
 		 */
 		UNIQUE,
-
```
Comment (rkorytkowski) [inline api/src/main/java/org/openmrs/PatientIdentifierType.java]: Why is this line removed?

Comment (tomaszmueller) [inline api/src/main/java/org/openmrs/PatientIdentifierType.java]: those are build stylistical changes
